src/components/sidebar/components/sidebar-back-to-link/index.tsx

```diff
@@ -1,17 +1,9 @@
 import { ReactElement } from 'react'
-import DevPopover from 'components/dev-popover'
 import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
-import MaybeInternalLink from 'components/maybe-internal-link'
-import Text from 'components/text'
+import StandaloneLink from 'components/standalone-link'
 import { SidebarBackToLinkProps } from './types'
 import s from './sidebar-back-to-link.module.css'
 
-const BackToLinkText: React.FC = ({ children }) => (
-  <Text asElement="span" size={200} weight="medium">
-    {children}
-  </Text>
-)
-
 /**
  * TODO: Design is still planning this functionality & states. Since not all
```

Comment:
Nit, non-blocking: I think we can remove this comment now that you've covered all the cases ☺️

Reply:
Oh, right! Yes I'll remove. 😊